src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -274,6 +276,19 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		});
 	}
 
+	public get viewModeChanged(): Event<ViewMode> {
+		return this._viewModeChangedEmitter.event;
+	}
+
+	public get viewMode() {
+		return this._viewMode;
+	}
+
+	public set viewMode(mode: ViewMode) {
+		this._viewMode = mode;
+		this._viewModeChangedEmitter.fire(mode);
```

Comment:
Probably only fire event if the mode has actually changed (mode !== this._viewMode)

Reply:
Talked to @chlafreniere. He agreed that the Views settings should go under the Notebooks section, so I moved it.